### PR TITLE
Format markdown files correctly

### PR DESCRIPTION
**Pull request by dustinsommerfeld**
Formatting markdown files correctly as specified in the style guide (issue #103).

### CODE REVIEW

**Comment by daniel-vera-g** [comment]
Nice @dustinsommerfeld , which formatter did you use? I know `markdownlint` or `prettier`. As the later can be used for ts too, maybe it would be more beneficial.
I would also add a `lint` or `format` command to the `package.json` to easily run `npm run lint/format` and get the files into the same format. It could also be added to the build stage of the ci/cd to ensure, that the content hast the proper format before deploying/merging.

Ah, and could we add the `.DS_Store` to the `.gitignore` ?

P.S: Can't _review_. properly as I'm not in the orga, maybe should ask to be added 😄

**Review by daniel-vera-g — CHANGES_REQUESTED** [review]
Siehe mein comment: https://github.com/fsiwi-hka/iwi-website/pull/104#issuecomment-1053536144

File: content/faq/infrastruktur.md [hunk 1]
```diff
@@ -7,17 +7,20 @@ header:
...
 ### Wie verhindere ich, dass Mails an die HS-Adresse verloren gehen?
-Das Greylisting des RZ wurde abgeschalten, somit sollten auch eMails von externen Adressen jetzt einfach weitergeleitet werden.
+
+Das Greylisting des RZ wurde abgeschalten, somit sollten auch Mails von externen Adressen jetzt einfach weitergeleitet werden.
 
 ### Dienste der Hochschule
+
 Die Hochschule selbst bietet uns Studierenden verschiedene Dienste wie z.B. Poolräume und verschiedenste Software an. Verantwortlich dafür ist das Rechenzentrum (RZ), auf dessen Website sich verschiedene Übersichten dieser Dienste befinden:
  - [Dienste speziell für Studierende](https://www.h-ka.de/rz)
  - [Übersicht über das gesamte Portfolio](https://www.hs-karlsruhe.de/servicekatalog)
```
Comment (mapelpapel) [inline content/faq/infrastruktur.md:26]:
```suggestion
Die Hochschule selbst bietet uns Studierenden verschiedene Dienste wie z.B. Poolräume und verschiedenste Software an. Verantwortlich dafür ist das Rechenzentrum (RZ), auf dessen Website sich verschiedene Übersichten dieser Dienste befinden: https://www.h-ka.de/rz
```
hs-karlsruhe.de is deprecated

File: content/pages/impressum.md [hunk 2]
```diff
@@ -5,68 +5,43 @@ header:
...
+
 ## Angaben gemäß § 5 TMG
 
-Fachschaft Informatik und Wirtschaftsinformatik<br />
-Moltkestraße 30<br />
+Fachschaft Informatik und Wirtschaftsinformatik  
+Moltkestraße 30  
 76133 Karlsruhe
 
 Vertreten durch: Zoe Kammerdiener, Anja Buchmaier
```
Comment (mapelpapel) [inline content/pages/impressum.md:15]:
```suggestion
Vertreten durch: Anja Buchmaier, Jens Ettl
```

File: content/pages/impressum.md [hunk 3]
```diff
@@ -5,68 +5,43 @@ header:
...
 
 [icon:at|E-Mail schreiben](/scripts/email.php?address=kontakt)
-
 [icon:phone|+49 721 925-1449](tel:+497219251449)
 
 ## Verantwortlich für den Inhalt nach § 55 Abs. 2 RStV
 
-Zoe Kammerdiener<br />
+Zoe Kammerdiener  
 Anja Buchmaier
```
Comment (mapelpapel) [inline content/pages/impressum.md:25]:
```suggestion
Anja Buchmaier  
Jens Ettl
```

**Review by mapelpapel — APPROVED** [review]
While formatting, we could also update / fix some content :)

**Comment by dustinsommerfeld** [comment]
> Nice @dustinsommerfeld , which formatter did you use? I know `markdownlint` or `prettier`. As the later can be used for ts too, maybe it would be more beneficial. I would also add a `lint` or `format` command to the `package.json` to easily run `npm run lint/format` and get the files into the same format. It could also be added to the build stage of the ci/cd to ensure, that the content hast the proper format before deploying/merging.
> 
> Ah, and could we add the `.DS_Store` to the `.gitignore` ?
> 
> ~P.S: Can't _review_. properly as I'm not in the orga, maybe should ask to be added 😄~

Hello @daniel-vera-g,
I don't use any formatting tool, I did it myself.
And the .DS_Store file is created by my macOS :-)

**Review by mapelpapel — DISMISSED** [review]

**Review by daniel-vera-g — APPROVED** [review]
Hab die .DS_Store in 1287958d83b8358b850861556e96b7f617ae9944 in die gitignore gepackt @dustinsommerfeld . Jetzt stört Sie in Zukunft nicht.

Wenn du das manuell gemacht hast passt das. Vlt. mache ich mal in Zukunft eine PR für einen markdown formatter oder so.

**Pull request merged**